Mono.Addins.Setup/Mono.Addins.Setup/RepositoryRegistry.cs

```diff
@@ -68,11 +68,12 @@ internal RepositoryRegistry (SetupService service)
 		/// The repository index is not downloaded by default. It can be downloaded
 		/// by calling UpdateRepository.
 		/// </remarks>
+		[Obsolete ("Use different overload")]
```

Comment:
No need to mark as obsolete. Just document that the default provider is MonoAddins.